src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_REGEX = "^[a-zA-z0-9.!#$%&'*+/=?^_`{|}~-]+@((\\[[0-9]{1,3}"
+            + "\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        return email != null && email.matches(EMAIL_REGEX);
+    }

```

Comment:
It is better to have the constant of a type `Pattern`, because otherwise a new object will be created each time you call method `matches`